examples/linear_extension/linear_extension.jl

```diff
@@ -127,19 +127,16 @@ function run_linear_extension_example()
     # point 1
     u_analytic_p₁ = u_ijk_analytic(load_factors(sa), p₁[1], p₁[2], p₁[3])
     uᵢ_analytic_p₁ = u_analytic_p₁[1]
-    uⱼ_analytic_p₁ = u_analytic_p₁[2]
-    uₖ_analytic_p₁ = u_analytic_p₁[3]
     # point 2
     u_analytic_p₂ = u_ijk_analytic(load_factors(sa), p₂[1], p₂[2], p₂[3])
     uᵢ_analytic_p₂ = u_analytic_p₂[1]
-    uⱼ_analytic_p₂ = u_analytic_p₂[2]
-    uₖ_analytic_p₂ = u_analytic_p₂[3]
     ## Strains
     "Computes strains numeric solution ϵᵢ, ϵⱼ and ϵₖ for analytic validation."
-    function ϵ_ijk_analytic(λᵥ::Vector{<:Real}, x₀::Real, y₀::Real, z₀::Real, ν::Real=ν, E::Real=E)
-        𝐶(t) = tension(t) * (1 - ν - 2ν^2) / (1 - ν)
+    function ϵ_ijk_analytic(λᵥ::Vector{<:Real}, x₀::Real, y₀::Real, z₀::Real, nu::Real=nu,
```

Comment:
Do you want to keep \epsilon and \sigma ?

Reply:
by now i think we can keep them since the main focus of this PR is to solve the visualization problems in the other unicode characters...